zhaquirks/tuya/ts0502b.py

```diff
@@ -0,0 +1,128 @@
+    #   "endpoints": {
+    #     "1": {
+    #       "profile_id": "0x0104",
+    #       "device_type": "0x010c",
+    #       "input_clusters": [
+    #         "0x0000",
+    #         "0x0003",
+    #         "0x0004",
+    #         "0x0005",
+    #         "0x0006",
+    #         "0x0008",
+    #         "0x0300",
+    #         "0x1000",
+    #         "0xef00"
+    #       ],
+    #       "output_clusters": [
+    #         "0x000a",
+    #         "0x0019"
+    #       ]
+    #     },
+    #     "242": {
+    #       "profile_id": "0xa1e0",
+    #       "device_type": "0x0061",
+    #       "input_clusters": [],
+    #       "output_clusters": [
+    #         "0x0021"
+    #       ]
+    #     }
+    #   },
+    #   "manufacturer": "_TZ3210_xwqng7ol",
+    #   "model": "TS0502B",
+    #   "class": "zigpy.device.Device"
+    # }
+    signature = {
+        MODELS_INFO: [
+            ("_TZ3210_xwqng7ol", "TS0502B"),
+            ("_TZ3210_frm6149r", "TS0502B"),
+            ("_TZ3210_jtifm80b", "TS0502B"),
+        ],
+        ENDPOINTS: {
+            # <SimpleDescriptor endpoint=1 profile=260 device_type=268
+            # input_clusters=[0, 3, 4, 5, 6, 8, 768, 4096, 61184]
+            # output_clusters=[10, 25]>
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.COLOR_TEMPERATURE_LIGHT,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    LevelControl.cluster_id,
+                    Color.cluster_id,
+                    LightLink.cluster_id,
+                    TuyaManufCluster.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [Time.cluster_id, Ota.cluster_id],
+            },
+            242: {
+                # <SimpleDescriptor endpoint=242 profile=41440 device_type=97
+                # input_clusters=[]
+                # output_clusters=[33]
+                PROFILE_ID: zgp.PROFILE_ID,
+                DEVICE_TYPE: zgp.DeviceType.PROXY_BASIC,
+                INPUT_CLUSTERS: [],
+                OUTPUT_CLUSTERS: [GreenPowerProxy.cluster_id],
+            },
+        },
+    }
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.COLOR_TEMPERATURE_LIGHT,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    LevelControl.cluster_id,
+                    LightLink.cluster_id,
+                    TuyaManufCluster.cluster_id,
```

Comment:
Remove the `.cluster_id` here. The Tuya manufacturer cluster is actually inside the ZCL range (and not the dedicated manufacturer range), so we can't just provide the `cluster_id`, but have to provide a (custom) cluster class.